File: Assignment_9_10_19/Python/odd_range.py
def OddRange(num1,num2):
    l = num1
    h = num2
    list= []
    for i in range(l,h):
        if i % 2 != 0:
            list.append(i)
    return list

x = OddRange(0,13)
print(x)

File: Assignment_9_10_19/JavaScript/oddrange.js
function OddRange(num1, num2) {
    var l = num1;
    var h = num2;
    for(i=l;i<h;i++) {
        if(i%2 !==0){
            console.log(i);
        }
    }
}
OddRange(3,13);
